src/commcare_cloud/commands/terraform/constants.py

```diff
@@ -6,6 +6,8 @@
 from __future__ import unicode_literals
 COMMCAREHQ_XML_POST_URLS_REGEX = r"""
 ^/a/([\w\.:-]+)/api/v([\d\.]+)/form/$
+^/a/([\w\.:-]+)/api/v0.6/case/$
+^/a/([\w\.:-]+)/api/v0.6/case/([\w-]+)/$
```

Comment:
@esoergel do you know why these weren't included here before? Did you add these routes to the WAF manually?

Reply:
Ah, I did not add these to the WAF manually, but I did decorate the view with `@waf_allow('XSS_BODY')`.  Is this a build step that's supposed to be done whenever that decorator is applied?

Reply:
@esoergel According to [this doc](https://confluence.dimagi.com/display/internal/How+traffic+gets+to+our+system%3A+Global+Accelerator%2C+ALB%2C+and+WAF) (see section _Allow-listing blocked traffic in the WAF_) this `constants.py` file has to be updated accordingly when using the decorator for a route view.

Reply:
@esoergel  I'm curious, is your view that's being decorated working for whatever reason you decorated it for? If you did not update this file and for argument's sake your changes works fine, I'm wondering if updating this file is technically necessary then?

Reply:
I'm not sure in what circumstances the WAF would block access to an undecorated view, but that view hasn't been released, so it's only been used lightly, in testing.